File: 003/003.dart
import "dart:io";

void main() {
  stdout.write("digite um numero: ");
  int? num1 = int.parse(stdin.readLineSync()!);
  stdout.write("digite outro numero: ");
  int? num2 = int.parse(stdin.readLineSync()!);
  print("a soma é ${num1 + num2}");
}

File: 003/003.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var readline = require("readline");
var rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
rl.question("digite um numero: ", function (num1) {
    rl.question("digite outro numero: ", function (num2) {
        var soma = parseInt(num1) + parseInt(num2);
        console.log("a soma dos dois numeros é: " + soma);
    });
});
